packages/static-build/test/build.test.ts

```diff
@@ -0,0 +1,54 @@
+import path from 'path';
```

Comment:
This file name is confusing next to `builds.test.ts`: https://github.com/vercel/vercel/tree/update/static-build-build-output-v3/packages/static-build/test

Are these the best names for these files?

Reply:
`builds.test.js` isn't even used apparently. It looks like it's unit tests though so probably should be merged into this new file. I'd prefer to do it in another PR.